csc/cmd/controller_list_snapshots.go

```diff
@@ -28,6 +30,8 @@ var listSnapshotsCmd = &cobra.Command{
 		req := csi.ListSnapshotsRequest{
 			MaxEntries:    listSnapshots.maxEntries,
 			StartingToken: listSnapshots.startingToken,
+			SnapshotId:listSnapshots.SnapshotId,
```

Comment:
Please follow the existing spacing conventions. This should be rewritten as:

```go
req := csi.ListSnapshotsRequest{
	MaxEntries:     listSnapshots.maxEntries,
        StartingToken:  listSnapshots.startingToken,
        SnapshotId:     listSnapshots.SnapshotId,
	SourceVolumeId: listSnapshots.sourceVolumeId,
}
```

Note how entires on the right-hand all line up together, and are one space offset from the longest variable name. Most go-editors correct this for you automatically.

Reply:
Thanks @Travis, Ya I missed formating. Now I formatted using GoLand.